brainzutils/cache.py

```diff
@@ -273,19 +253,15 @@ def gen_key(key, *attributes):
 
 def _prep_dict(dictionary, namespace=None, encode=True):
     """Wrapper for _prep_key and _encode_val functions that works with dictionaries."""
-    if namespace:
-        namespace_and_version = _append_namespace_version(namespace)
-    else:
-        namespace_and_version = None
-    return {_prep_key(key, namespace_and_version): _encode_val(value) if encode else value
+    return {_prep_key(key, namespace): _encode_val(value) if encode else value
             for key, value in dictionary.items()}
 
 
-def _prep_key(key, namespace_and_version=None):
+def _prep_key(key, namespace):
     """Prepares a key for use with Redis."""
     # TODO(roman): Check if this is actually required for Redis.
-    if namespace_and_version:
-        key = "%s:%s" % (namespace_and_version, key)
+    if namespace:
+        key = "%s:%s" % (namespace, key)
     if not isinstance(key, bytes):
         key = key.encode(ENCODING_ASCII, errors='xmlcharrefreplace')
```

Comment:
we talked about this in irc, right? I think the idea was that we'd leave keys as strings, instead of encoding to bytes? But then we'd also ensure that they were ascii only

Reply:
Yes, I have done this in #61.